### PR TITLE
add nightly build ID

### LINKED ISSUE #140: standalone binaries built with CI don't actually work
```
λ floss.exe
Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth__tkinter.py", line 28, in <module>
IOError: Tcl data directory "C:\Users\willi\AppData\Local\Temp\_MEI16~1\tcl" not found.
Failed to execute script pyi_rth__tkinter
```

### PR DESCRIPTION
**Pull request by williballenthin**
when building standalone exes with appveyor or travis, embed the current git commit in the version string.

also fix the resources bundled with the standalone exe. closes #140.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ install:
     - "%PYTHON%\\python.exe --version"
     - "%PYTHON%\\Scripts\\pip install https://github.com/williballenthin/vivisect/zipball/master"
     - "%PYTHON%\\Scripts\\pip install pyinstaller"
+    - ps: Set-Content "floss\\version.py" ( "__version__ = '{0}'" -f ( git describe --tags --always ) )
```
Comment (williballenthin) [inline .appveyor.yml]: looks like: `floss.exe 9e4823c`

File: .travis.yml [hunk 2]
```diff
@@ -11,6 +11,7 @@ env:
 install:
     - pip install https://github.com/williballenthin/vivisect/zipball/master
     - pip install pyinstaller
+    - echo "__version__ = '$(git describe --tags)'" > floss/version.py
```
Comment (williballenthin) [inline .travis.yml]: looks like: `floss.1 v1.1.0-140-g2572f57`

File: floss.spec [hunk 3]
```diff
@@ -124,11 +124,19 @@ a = Analysis(
...
              cipher=block_cipher)
 
+a.binaries = a.binaries - TOC([
+ ('sqlite3.dll', None, None),
+ ('tcl85.dll', None, None),
+ ('tk85.dll', None, None),
+ ('_sqlite3', None, None),
+ ('_ssl', None, None),
+ ('_tkinter', None, None)])
+
```
Comment (williballenthin) [inline floss.spec]: not sure which of these two changes actually fixes #140, but i think its the first one. the second one ensures there aren't extra DLLs included as resources.

Comment (mr-tz) [inline floss.spec]: Awesome! Thanks for figuring this out.

**Comment by mr-tz** [comment]
Looks great!

**Pull request merged**